config/src/main/java/com/quorum/tessera/config/util/EnvironmentVariableProvider.java

```diff
@@ -15,4 +15,10 @@ public String getEnv(String name) {
             .orElse(null);
     }
 
+    public boolean hasEnv(String name) {
+        Optional<String> env = Optional.ofNullable(System.getenv(name));
```

Comment:
Seems a bit much for a null check. Is `return Objects.isNull(getEnv(name));` enough?

Reply:
I have kept this as-is for the time being as it is a workaround for the jacoco coverage report.  `System.getenv()` cannot be mocked and it is not trivial to set the environment variables when testing.  If we were to use `Objects.isNull(...)`, then both the `null` and non-`null` cases would need to be tested to satisfy jacoco.  Using the Optional means we achieve coverage by only testing the `null` case.